src/Jackalope/Transport/Jackrabbit/curl.php

```diff
@@ -68,11 +68,11 @@ public function setopt_array(array $options): bool
     /**
      * Performs a cUrl session.
      *
-     * @return bool false on failure otherwise true or string (if CURLOPT_RETURNTRANSFER option is set)
+     * @return bool|string false on failure otherwise true or string (if CURLOPT_RETURNTRANSFER option is set)
      *
      * @see curl_exec
      */
-    public function exec(): bool
```

Comment:
this silently converted the string to boolean, in the process destroying the response :facepalm: